### ISSUE TITLE
[Bug]: Enchanting Sweeping Edge enables friendly fire.

### ISSUE THREAD

**Issue by Neodymiump** [Fuzss/spikyspikes#2]
### Mod Loader (Required)

Forge

### Minecraft Version (Required)

v1.20.1

### Mod Version (Required)

v8.0.0

### Notes (Required)

Occurred on a forge server.
I have not yet tested it without enchantment, but will report it as soon as possible.

**Comment by Fuzss** [comment]
Please provide more details. In what capacity does it enable friendly fire, who is hurt under what circumstances and what‘s the expected behaviour.

**Comment by Neodymiump** [comment]
I have created a trap tower, it is a simple one that collects in a stream of water and handles manually.
Instead of handling it manually, I placed the spikes and I am damaged at the Sweeping Edge where the spikes are triggered.
It is not every time, it is a probability. But not less. :(

![image](https://github.com/Fuzss/spikyspikes/assets/19851409/63b49985-ba60-4998-ab83-9b24d38ecaf3)
![image](https://github.com/Fuzss/spikyspikes/assets/19851409/2f5eb433-761e-4b09-90a3-7d747c6afa3a)
